app/decorators/tweet_decorator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'twitter'
+
+class TweetDecorator
+  HASHTAG = '#bienenlive'
+  HOST = 'https://bienenlive.wdr.de'
+
+  def initialize(diary_entry)
+    @diary_entry = diary_entry
+  end
+
+  def tweet!
+    client = Twitter::REST::Client.new do |config|
+      config.consumer_key = ENV['TWITTER_CONSUMER_KEY']
+      config.consumer_secret = ENV['TWITTER_CONSUMER_SECRET']
+      config.access_token = ENV['TWITTER_ACCESS_TOKEN']
+      config.access_token_secret = ENV['TWITTER_ACCESS_TOKEN_SECRET']
+    end
+
+    client.update(tweet_content)
+  end
+
+  def read_more_link
+    report = @diary_entry.report
+    return HOST if report.frontend_base_url.blank?
+
+    "#{HOST}/#{report.frontend_base_url}/mein-tagebuch/#{@diary_entry.id}"
```

Comment:
Es ist nicht sicher, ob der Slug `mein-tagebuch` ist.

```suggestion
    "#{HOST}/#{report.frontend_base_url}/#{@diary_entry.id}"
```

```suggestion
    "#{HOST}/#{report.frontend_base_url}/#{DIARY_SLUG}/#{@diary_entry.id}"
```